cypress/support/helpers/materialsCollectionsHelper.js

```diff
@@ -0,0 +1,23 @@
+export function verifyMaterialsCollectionPage(){
+  cy.get('#content .action_menu_header .action_menu_header_right a').contains("create Materials Collection");
+}
+
+export function verifyFilterResult(count){
+  cy.get('.action_menu_header .action_menu_header_left p').contains(count);
```

Comment:
"count" is a misleading name for a string arg: I assumed this had to be an integer, until I saw how it was used in admin_materials_collections_filter.spec.js. How about something like "filterResult", or "countDescription?